openlibrary/components/BulkSearch/utils/classes.js

```diff
@@ -8,13 +8,82 @@ export class ExtractedBook {
     }
 }
 
+class AbstractExtractor {
+    constructor(name = '') {
```

Comment:
This is required, so let's not give it a default
```suggestion
    constructor(name) {
```

Reply:
Defaults removed.